release/models/policy/openconfig-routing-policy.yang

```diff
@@ -54,7 +54,13 @@ module openconfig-routing-policy {
     corresponding action statement is executed.  If the action
     statement has either accept-route or reject-route actions, policy
     evaluation of the current policy definition stops, and no further
```

Comment:
```suggestion
    evaluation of the current policy definition stops after having executed all the defined actions , and no further policy statements are evaluated. In case there is a policy chain, no further 
```

Reply:
I like this and made a modification that is very similar to what you suggest.